fgallery

```diff
@@ -138,82 +124,74 @@ sub is_fgallery_dir
 sub slurp
 {
   my ($fn) = @_;
-  open(my $fd, '<', $fn) or fatal("cannot read $fn: $!");
-  if ($^V lt v5.23.4) {
-    binmode($fd, ":encoding($ENCODING)");
-  }
+  no warnings; # avoid wide character warnings
```

Comment:
Is there no way to get rid of the wide character warnings by using the encoding layer in the open call?
> open(my $fd, '<:raw', $fn) or simply default to utf8?